rocAL/rocAL/include/pipeline/master_graph.h

```diff
@@ -113,6 +113,7 @@ class MasterGraph
     TensorList * labels_meta_data();
     TensorList * bbox_meta_data();
     TensorList * mask_meta_data();
+    TensorList * pixelwise_meta_data();
```

Comment:
I feel this API is not necessary
We can use the API for mask meta data itself, Since both are doing the same operations.
Also user can enable either pixelwise or polygon masks, So we can use the mask_meta_data API itself and I feel a separate API is not required for this in master graph